src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1389,6 +1389,206 @@ public async Task TestRuntimeBaseRouteInNextLinkForPaginatedRestResponse()
+            File.WriteAllText(CUSTOM_CONFIG, configuration.ToJson());
+            string[] args = new[]
+            {
+                $"--ConfigFileName={CUSTOM_CONFIG}"
+            };
+
+            using (TestServer server = new(Program.CreateWebHostBuilder(args)))
+            using (HttpClient client = server.CreateClient())
+            {
+                HttpMethod httpMethod = SqlTestHelper.ConvertRestMethodToHttpMethod(SupportedHttpVerb.Post);
+                HttpRequestMessage request = new(httpMethod, requestPath);
+                if (entityType is not EntitySourceType.StoredProcedure)
+                {
+                    string requestBody = @"{
+                        ""title"": ""Harry Potter and the Order of Phoenix"",
+                        ""publisher_id"": 1234
+                    }";
+
+                    JsonElement requestBodyElement = JsonDocument.Parse(requestBody).RootElement.Clone();
+                    request = new(httpMethod, requestPath)
+                    {
+                        Content = JsonContent.Create(requestBodyElement)
+                    };
+                }
+
+                HttpResponseMessage response = await client.SendAsync(request);
+
+                // Location header field is expected only when POST request results in the creation of a new item
+                Assert.AreEqual(HttpStatusCode.Created, response.StatusCode);
+
+                string locationHeader = response.Headers.Location.AbsoluteUri;
+
+                // GET request performed using the Location header should be successful.
+                HttpRequestMessage followUpRequest = new(HttpMethod.Get, response.Headers.Location);
+                HttpResponseMessage followUpResponse = await client.SendAsync(followUpRequest);
+                Assert.AreEqual(HttpStatusCode.OK, followUpResponse.StatusCode);
+
+                // Delete the new record created as part of this test
+                if (entityType is EntitySourceType.Table)
+                {
+                    HttpRequestMessage cleanupRequest = new(HttpMethod.Delete, locationHeader);
+                    await client.SendAsync(cleanupRequest);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Validates the Location header field returned for a POST request when it results in a 201 response. The idea behind returning
+        /// a Location header is to provide a URL against which a GET request can be performed to fetch the details of the new item.
+        /// Base Route is configured in the config file used for this test. So, it is expected that the Location header returned will contain the base-route.
+        /// This test performs a POST request, and checks if it results in a 201 response. If so, the test validates the correctness of the Location header in two steps.
+        /// Since, base-route has significance only in the SWA-DAB integrated scenario and this test is executed against DAB running independently,
```

Comment:
```suggestion
        /// Since base-route has significance only in the SWA-DAB integrated scenario and this test is executed against DAB running independently,
```